coderd/database/queries/provisionerdaemons.sql

```diff
@@ -35,7 +35,7 @@ VALUES (
 	@last_seen_at,
 	@version,
 	@api_version
-) ON CONFLICT("name", lower((tags ->> 'owner'::text))) DO UPDATE SET
+) ON CONFLICT("name", lower(COALESCE(tags ->> 'owner'::text, ''::text))) DO UPDATE SET
```

Comment:
Question: out of a consistency perspective, should we try to match the format for index/query as closely as possible? If going by migration, it'd be `lower(coalesce(tags->>'owner', '')::text)`.

Reply:
At this point, given how complex the `ON CONFLICT` clause is becoming, I'd be much more inclined to say `ON CONFLICT idx_provisioner_daemons_name_owner_key` instead. WDYT?

Reply:
Oh yeah, for sure if it works. I have a recollection of that not always being possible so I didn't think to question it 😄

Reply:
Turns out you can't do that in this specific instance 😢